entrypoint.sh

```diff
@@ -337,10 +346,15 @@ function osd_directory {
     fi
 
     mkdir -p /etc/service/${CLUSTER}-${OSD_ID}
-    echo "store-daemon: starting daemon on ${HOSTNAME}..."
-    exec ceph-osd ${CEPH_OPTS} -f -d -i ${OSD_ID} --osd-journal ${OSD_J} -k /var/lib/ceph/osd/${CLUSTER}-${OSD_ID}/keyring
-
+    cat >/etc/service/${CLUSTER}-${OSD_ID}/run <<EOF
+#!/bin/bash
+echo "store-daemon: starting daemon on ${HOSTNAME}..."
+exec ceph-osd ${CEPH_OPTS} -f -d -i ${OSD_ID} --osd-journal ${OSD_J} -k /var/lib/ceph/osd/${CLUSTER}-${OSD_ID}/keyring
+EOF
```

Comment:
OK I will remove this. That means we probably shouldn't have the previous mkdir line either.